README.rst

```diff
@@ -572,28 +573,28 @@ Here is an example of ``.cartridge.yml``:
     script: my-init.lua
     stateboard: true
 
-**Note:** the config of the `standard application template <Creating an application from template_>`_ initially has the ``stateboard: true`` parameter.
+**Note:** the config of the `standard application template <Creating an application from template_>`_
+initially has the ``stateboard`` parameter set to ``true``.
 
-.. // Please, update the doc in cli/commands on updating this section
+.. // Please update the doc in cli/commands on updating this section
 
-*********
+********
 ``stop``
-*********
+********
 
-To stop one or more running instances, say:
+To stop one or more running instances, execute:
```

Comment:
we usually use 'run'.

Reply:
"...running instances, run" looks less than ideal